mod/ckeditor/views/default/ckeditor/init.php

```diff
@@ -9,7 +9,7 @@
 }
 ?>
 <script>
-	require(['elgg/ckeditor'], function (elggCKEditor) {
+	require(['elgg/ckeditor/config', 'ckeditor/ckeditor', 'elgg/ckeditor'], function(elggCKEditor) {
```

Comment:
elgg/ckeditor already depends on `elgg/ckeditor/config` and `ckeditor` (defined in views.php), so this change doesn't do anything, but loads the CKeditor library twice, once as `ckeditor.js` and once as `ckeditor/ckeditor.js`.